Input Search Dark Mode Friendly

THEBOSS0369 opened the pull request:
This PR fixes #5734

**Description**

I have added some changes so that the input looks like more dark friendly. The changes for style has been done for **Intergration Page**, **Home Page** and **Blog Page**. The style is same as the style in **layer5/news** page. Below are the ss for the changes made.

![main page](https://github.com/user-attachments/assets/8e4d79b7-a185-4165-8c07-938fb9e8980a)
![main page](https://github.com/user-attachments/assets/e6b00146-5caa-49ed-89cc-466e87b44603)
![blog](https://github.com/user-attachments/assets/72e4a294-b7d9-4261-ac38-08a8fb0037b3)

**Notes for Reviewers**
This PR is same as this one https://github.com/layer5io/layer5/pull/5881 . It is made because of DCO error not resolving.


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

l5io commented:
🚀 Preview for commit ae8e3af6ad52218ab114f1fec426514e79edf407 at: https://66e6cb30f79a9a88c715ccb7--layer5.netlify.app

Ashparshp commented:
@THEBOSS0369, Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

I'm adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk) || [meeting link](https://meet.layer5.io/websites).

vishalvivekm commented on src/theme/app/themeStyles.js:
We wouldn't want to change theme colors @THEBOSS0369

vishalvivekm commented:
Also, will you please remove commits not pertinent to the issue ?

THEBOSS0369 commented:
> @THEBOSS0369 
> Sharing the links form our discussion in [websites meeting](https://www.youtube.com/watch?v=7ln7cWOHOKo&t=170s) 
> - https://github.com/layer5io/layer5/blob/b0d781f24c3f323b96a772c61b8446a291896cb2/src/sections/Company/News-grid/NewsGrid.style.js#L102
> - https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#things-to-keep-in-mind-1

Thanks for sharing this!! 🙂

THEBOSS0369 commented:
> @THEBOSS0369 
> Sharing the links form our discussion in [websites meeting](https://www.youtube.com/watch?v=7ln7cWOHOKo&t=170s) 
> - https://github.com/layer5io/layer5/blob/b0d781f24c3f323b96a772c61b8446a291896cb2/src/sections/Company/News-grid/NewsGrid.style.js#L102
> - https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#things-to-keep-in-mind-1

Thanks for sharing this!! 🙂🙂

THEBOSS0369 commented:
> Also, will you please remove commits not pertinent to the issue ?

Sure! I will remove the changes from searchbox. Js and apply the changes in gridestyle. Js

vishalvivekm commented:
> > Also, will you please remove commits not pertinent to the issue ?
> 
> Sure! I will remove the changes from searchbox. Js and apply the changes in gridestyle. Js

I was referring to these Commits: 
![Screenshot_20240917-084243](https://github.com/user-attachments/assets/5c6c88ea-fdeb-42e7-8c21-38a1689b2cce)

leecalcote commented on src/reusecore/Search/searchbox.style.js:
```diff
@@ -15,6 +15,10 @@ export const SearchWrapper = styled.div`
       background-color: ${(props) => props.theme.shadowLightColor};
+      
+      /*Changes color for the text*/
+      color: ${(props) => props.theme.textColor};
```
Will this affect the search field on /blog, too?

Hopefully, it does as that field is in need of similar dark mode treatment.

leecalcote replied:
I see now that it does.

THEBOSS0369 commented:
> > > Also, will you please remove commits not pertinent to the issue ?
> > 
> > Sure! I will remove the changes from searchbox. Js and apply the changes in gridestyle. Js
> 
> I was referring to these Commits: 
> ![Screenshot_20240917-084243](https://github.com/user-attachments/assets/5c6c88ea-fdeb-42e7-8c21-38a1689b2cce)
> 

Ohhhh, i get it now! I didn't commited those changes but can you tell me how do i remove those commits like do i have to revert back or any other way.

THEBOSS0369 commented:
@vishalvivekm Can you confirm that i have to do this right?? 
![image](https://github.com/user-attachments/assets/dff2d527-c190-40df-a984-0f1ec9164072)

THEBOSS0369 commented:
Hey @vishalvivekm ! Please review the changes i made. Also, now the style is similar as the style of searchbox in /news page.

l5io commented:
🚀 Preview for commit b263459fbbe55c1764dbfddf18c496752daebcee at: https://66ed04ee857b09e66702c1f3--layer5.netlify.app

vishalvivekm commented:
@THEBOSS0369 why are we still modifying the themes hex codes?

vishalvivekm commented:
> @THEBOSS0369 why are we still modifying the themes hex codes?

Please review the feedback: https://www.youtube.com/watch?v=7ln7cWOHOKo&t=170s

THEBOSS0369 commented:
Hey @vishalvivekm ! Apologies for late reply. 
I got confused and wasn't able to figure out how. Now I have sorted everything, I have added the style for Searchbox in Required Page's   ...grid.style.js file and removed the changes from the Themestyles.js and searchbox.style.js . 

Note the Changes you will see in Searchhox.style.js are the changes which I copied from the official searchbox.style.js to revert back to the original color. 

Thanks

l5io commented:
🚀 Preview for commit df81b5ec70b776de3fdd1eed81111f7332496df7 at: https://66f5bc40a708be4127521040--layer5.netlify.app

sudhanshutech commented:
@THEBOSS0369 any updates?

THEBOSS0369 commented:
> @THEBOSS0369 any updates?

Hey @sudhanshutech ! This PR is completed and ready to merge. I had exams which ended today so i wasn't active on the slack. 
Thanks

sudhanshutech commented:
@THEBOSS0369 there was gatsby upgrade in site please ensure you changes again that it works.

l5io commented:
🚀 Preview for commit defd7355469a972c7a5ecd23f78633d6850df5d7 at: https://6706648dbcd8ce273bf87ec6--layer5.netlify.app

THEBOSS0369 commented:
@sudhanshutech ! Everything is working fine i just checked it here are ss. You can also verify these changes on the preview link.

![image](https://github.com/user-attachments/assets/a71e8a78-495a-45f3-93d6-b747a1cb29e5)
![image](https://github.com/user-attachments/assets/c8f00c86-c0c0-45a1-82b1-5011c67b307e)
![image](https://github.com/user-attachments/assets/b3ddc086-2f21-4a5b-ae67-5df612ca01f5)

l5io commented:
🚀 Preview for commit df8b9487c0aa0c2259754e8f89404a76a1bfe03c at: https://67066a3e750f902dccf909cf--layer5.netlify.app

sudhanshutech reviewed (changes requested):
![{ACFBDB30-87EE-48D7-9355-82A42F85CF69}](https://github.com/user-attachments/assets/4321a178-f538-46bb-ae21-3c18f840fffc)
this search icon should also be a white shade as well

sudhanshutech commented:
> > ![{ACFBDB30-87EE-48D7-9355-82A42F85CF69}](https://private-user-images.githubusercontent.com/90546692/376264099-4321a178-f538-46bb-ae21-3c18f840fffc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5MTUwMjAsIm5iZiI6MTcyODkxNDcyMCwicGF0aCI6Ii85MDU0NjY5Mi8zNzYyNjQwOTktNDMyMWExNzgtZjUzOC00NmJiLWFlMjEtM2MxOGY4NDBmZmZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDE0MDUyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmNjFhM2VjMDhlYTkwZWI4YjRlODgzOTQ4Mjc1ZmYyOTAyMjljOWVjMDE4MzE5YTczY2FjODQ1MTZkOGM0M2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uPP4VVlR2a__N2LG3cXSpvSAngkZMZlIHiX_oCcO8Es)
> > this search icon should also be a white shade as well
> 
> @sudhanshutech ! The style is same as in Layer5/news section and @vishalvivekm told me to copy the same style. Although I agree that it should be white. If you both agree to that I will change it to white.

yes please , for light mode it could stay the same but for dark mode a white shade. See if there other icons which do the same like dark shade for light mode and white shade for light mode and do the same

THEBOSS0369 commented:
@sudhanshutech @vishalvivekm ! I have removed the code from the required searchbox's respective files and add it to the global styles so that in future we don't have to copy paste the whole code.
 **Note**: I have also removed it from the News Section as well . 
Here are the Results!

![Screenshot 2024-10-17 121657](https://github.com/user-attachments/assets/7cdd480e-9723-44a6-9c1a-ef7d8be53f72)
![Screenshot 2024-10-17 121755](https://github.com/user-attachments/assets/8bd97f04-03ed-4303-9591-0136d537558f)
![Screenshot 2024-10-17 121831](https://github.com/user-attachments/assets/63cd4fc9-b6a1-4fcc-aec1-3dd9968fc8ba)
![Screenshot 2024-10-17 121912](https://github.com/user-attachments/assets/3498be27-1526-426d-9f75-59ba0ff9dad2)
![Screenshot 2024-10-17 122034](https://github.com/user-attachments/assets/d7a3caa8-3ee9-4d9f-92dd-c7f8df4f267d)

l5io commented:
🚀 Preview for commit 26dd7b6a429bf8bf7cf94cf552986fd246027c88 at: https://67134b3657a13b407c38a272--layer5.netlify.app

l5io commented:
🚀 Preview for commit 68842882b272557c634b23a7f729eb58ec2b311b at: https://671632d73284890bdad28f2c--layer5.netlify.app

THEBOSS0369 commented:
Hey @sudhanshutech @vishalvivekm ! I have done the changes you asked. Please look into it and i hope this is the last thing i needed to change. Thanks

![image](https://github.com/user-attachments/assets/b48158d5-14fe-4600-a9c5-e280dde7676e)
![image](https://github.com/user-attachments/assets/9df8f2a9-5f61-4f1f-ae2c-2cf4d3725c77)

l5io commented:
🚀 Preview for commit 433f23a42343cf140f1a0d58be12ee82ba6b0810 at: https://67179bdbb7a10f10183228ff--layer5.netlify.app

l5io commented:
🚀 Preview for commit 59368e32dfc09284a1faa0a3c84a72d33b6467da at: https://671ccd0237fc879c50759566--layer5.netlify.app

THEBOSS0369 commented:
> @THEBOSS0369 build is failing

@sudhanshutech This issue is not from my side. You can clearly see the build previews link before yesterday. I also checked other PR's tests the build test is failing in those PRs as well.

THEBOSS0369 commented:
@vishalvivekm I have moved the code for integration search bar to its own file as you asked me. Please let me know if any changes are needed! Here are the results 

![image](https://github.com/user-attachments/assets/2f7cf44b-15fe-46a3-879f-3282ece57d97)
![image](https://github.com/user-attachments/assets/c5fa90f0-4708-4964-8928-b5640244212f)

vishalvivekm commented:
> > @THEBOSS0369 build is failing
> 
> @sudhanshutech This issue is not from my side. You can clearly see the build previews link before yesterday. I also checked other PR's tests the build test is failing in those PRs as well.

should be fixed now

l5io commented:
🚀 Preview for commit ae1c6698fca3aa8f70a7b659a1d72724ff8cba23 at: https://6722282a049c71acb7dc5135--layer5.netlify.app

vishalvivekm commented:
Thanks much @THEBOSS0369 for your contributions.

THEBOSS0369 commented:
@vishalvivekm ! Thanks for merging this, now i can work on more issues 😁
